packages/kbn-investigation-shared/src/rest_specs/create.ts

```diff
@@ -19,6 +19,7 @@ const createInvestigationParamsSchema = t.type({
       timeRange: t.type({ from: t.number, to: t.number }),
     }),
     origin: t.union([alertOriginSchema, blankOriginSchema]),
+    tags: t.array(t.string),
```

Comment:
I think `tags` should be optional

Reply:
> We should also change filtering to use status: triage or status: active [here](https://github.com/elastic/kibana/blob/233fe603b523cfed600528c77fc04e135582a96c/x-pack/plugins/observability_solution/investigate_app/server/services/find_investigations.ts#L35). This filter is used to determine if there is any existing investigation for the alert.

yes good catch I need to change that using some types so it is covered at compile time.